### ISSUE TITLE
We need to strip out unneeded sections from the sphinx docs

### ISSUE THREAD

**Issue by timlinux** [AIFDR/inasafe-doc#275]
General, RoadMap and User Docs sections need to be removed from sources in inasafe-doc and front index page should be an index for training docs. Old index page can be ditched. Changes pushed to develop.

**Comment by rduivenvoorde** [comment]
Partly done:
https://github.com/AIFDR/inasafe-doc/tree/develop/docs/source
only user-docs are still there?

**Comment by timlinux** [comment]
Yes keep user docs and training there for now please as you have it. Thanks @rduivenvoorde

**Comment by rduivenvoorde** [comment]
@timlinux should I 'sign off' this issues myself, or is it you or another 'reviewer' to do this?
